packages/cli/src/commands/init/protocols.js

```diff
@@ -0,0 +1,37 @@
+// @flow
+import path from 'path';
+
+const FILE_PROTOCOL = /file:/;
```

Comment:
so we use `file:`, shouldn't it be `file://`?

Reply:
I should probably use https://nodejs.org/api/url.html.

Reply:
Since API is supported from `v10` of node, I will use https://www.npmjs.com/package/whatwg-url